### PR TITLE
Add portable build definitions

### PR DESCRIPTION
**Pull request by gkhanna79**
skip ci please

@weshaggard PTAL

### CODE REVIEW

File: buildpipeline/pipeline.json [hunk 1]
```diff
@@ -283,6 +311,75 @@
           }
         },
         {
+          "Name": "DotNet-CoreFx-Trusted-Windows",
+          "Parameters": {
+            "PB_Platform": "arm",
+            "PB_BuildArguments": "-buildArch=arm -Release -portable -- /p:SignType=real /p:RuntimeOS=win10",
```
Comment (weshaggard) [inline buildpipeline/pipeline.json]: I know you are just following the existing pattern but is probably better to use `$(PB_Platform)` in these other parameters (assuming that works).

File: buildpipeline/pipeline.json [hunk 2]
```diff
@@ -283,6 +311,75 @@
...
+            "PB_BuildTestsArguments": "-buildArch=arm -Release -SkipTests -Outerloop -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
```
Comment (weshaggard) [inline buildpipeline/pipeline.json]: Not 100% certain but you may need to pass `-portable` to the buildtests as well.

Comment (gkhanna79) [inline buildpipeline/pipeline.json]: What's a good way to validate whether tests need to be passed portable or not?

Comment (gkhanna79) [inline buildpipeline/pipeline.json]: @weshaggard any thoughts on this?

Comment (weshaggard) [inline buildpipeline/pipeline.json]: I think the best way to test this is to do the steps locally. Do a build.cmd -portable and then do a build-tests.cmd without portable. There is a chance that the non-portable assets will end up in the shared framework folder we are testing against.

**Comment by gkhanna79** [comment]
@weshaggard PTAL - changes per our discussion.

**Comment by gkhanna79** [comment]
@weshaggard Can you PTAL? I would like to merge it soon and get the builds going.

**Pull request merged**